joblib/numpy_pickle.py

```diff
@@ -224,6 +224,7 @@ def __init__(self, fp, protocol=None):
         if protocol is None:
             protocol = (pickle.DEFAULT_PROTOCOL if PY3_OR_LATER
                         else pickle.HIGHEST_PROTOCOL)
+        self.protocol = protocol
```

Comment:
I meant you can remove this line and the protocol is already kept as the `.proto` attribute.